ngmix/prepsfmom.py

```diff
@@ -0,0 +1,639 @@
+import logging
+
+import numpy as np
+import scipy.fft as fft
+
+from ngmix.observation import Observation
+from ngmix.moments import fwhm_to_sigma
+from ngmix.util import get_ratio_error
+from ngmix.gexceptions import FFTRangeError
+from ngmix.fastexp_nb import FASTEXP_MAX_CHI2, fexp_arr
+
+
+logger = logging.getLogger(__name__)
+
+
+class _PrePSFMom(object):
```

Comment:
Why hide this?

Using the string keyword makes programmatic code easier, e.g. a config parameter could be set to the string 'ksigma' or 'gauss' and passed directly.

Reply:
I didn't want to have to set a default but I can simply leave it free as the second argument.

Reply:
fixed